### PR TITLE
system(U5) update STM32U5xx HAL Drivers to v1.1.0 ,  CMSIS Drivers to v1.1.0

### PR DESCRIPTION
**Pull request by ABOSTM**
system(U5) update STM32U5xx HAL Drivers to v1.1.0
system(U5): update STM32U5xx CMSIS Drivers to v1.1.0

core(U5): update wrapped files

### CODE REVIEW

File: cores/arduino/stm32/stm32_def_build.h [hunk 1]
```diff
@@ -414,6 +414,14 @@
...
   #elif defined(STM32U585xx)
     #define CMSIS_STARTUP_FILE "startup_stm32u585xx.s"
+  #elif defined(STM32U595xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u595xx.s"
+  #elif defined(STM32U599xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u599xx.s"
+  #elif defined(STM32U5A5xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u5a5xx.s"
+  #elif defined(STM32U5A9xx)
+    #define CMSIS_STARTUP_FILE "startup_stm32u5a9xx.s"
```
Comment (fpistm) [inline cores/arduino/stm32/stm32_def_build.h:424]: Like there is new mcu. The variant for U5 should be regenerated to take then in count.

File: system/Drivers/STM32U5xx_HAL_Driver/Inc/stm32u5xx_hal_conf_template.h [hunk 2]
```diff
@@ -277,6 +286,10 @@ vary depending on the variations in voltage and temperature.*/
 #include "stm32u5xx_hal_dma2d.h"
 #endif /* HAL_DMA2D_MODULE_ENABLED */
 
+#ifdef HAL_DSI_MODULE_ENABLED
```
Comment (fpistm) [inline system/Drivers/STM32U5xx_HAL_Driver/Inc/stm32u5xx_hal_conf_template.h:289]: https://github.com/stm32duino/Arduino_Core_STM32/blob/main/system/STM32U5xx/stm32u5xx_hal_conf_default.h
should be updated to add all the new HAL modules.

**Review by fpistm — CHANGES_REQUESTED** [review]
In addition to the review comments:
- Fix merge conflict
- Fix build issue

File: cores/arduino/stm32/stm32_def_build.h (hunk 1 above)
Comment (ABOSTM) [reply cores/arduino/stm32/stm32_def_build.h:424]: Done

File: system/Drivers/STM32U5xx_HAL_Driver/Inc/stm32u5xx_hal_conf_template.h (hunk 2 above)
Comment (ABOSTM) [reply system/Drivers/STM32U5xx_HAL_Driver/Inc/stm32u5xx_hal_conf_template.h:289]: Done

**Comment by ABOSTM** [comment]
> In addition to the review comments:
> 
> * Fix merge conflict
> * Fix build issue

Done

**Pull request merged**